pkg/common/scheduler/scheduler.go

```diff
@@ -40,6 +40,8 @@ type Evictor func(vpod VPod, from *duckv1alpha1.Placement) error
 type Scheduler interface {
 	// Schedule computes the new set of placements for vpod.
 	Schedule(vpod VPod) ([]duckv1alpha1.Placement, error)
+	GetVReplicaMPS() int32
+	GetMaxEventPerSecondPerPartition() int32
```

Comment:
the scheduler is a generic component that does not know anything about Kafka. Please remove.

Reply:
How do I pass the two new env vars to `pkg/source/reconciler/mtsource/kafkasource.go`?